### PR TITLE
feat: rework some prompts

### PR DESCRIPTION
**Pull request by Sparkier**

### CODE REVIEW

**Comment by cabreraalex** [comment]
Do we want a persona editor? Do you think it makes sense in many cases? I don't know... maybe it does

**Comment by Sparkier** [comment]
> Do we want a persona editor? Do you think it makes sense in many cases? I don't know... maybe it does

Not so sure myself tbh. I tested with sample size 1 and found it useful but maybe we test more. I don't know generally how we do this. Maybe we hide them behind a feature flag per default and only show ones we know are useful for other people?

**Comment by cabreraalex** [comment]
I like that idea! Let's merge for now but in another PR add feature flags for each editor?

File: src/lib/server/editors/PersonaEditor.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+			label: string;
+			predictions: { prediction: string }[];
+		}[],
+		input: string | unknown
+	): Promise<Tables<'prompts'>> {
+		const systemPrompt = `You are an AI assistant that rewrites prompts to include a description of the persona of the AI model. Users provide a prompt and a persona description. Your task is to add a description of the persona to the prompt.
+
+### Instructions
+* You do not modify the prompt in any other way. Specifically the general instruction AND formatting of the propmt should not be changed. 
+* Make sure the desired persona is added somewhere towards the end of the prompt.
```
Comment (cabreraalex) [inline src/lib/server/editors/PersonaEditor.ts:73]: maybe the start of the prompt?

**Review by cabreraalex — APPROVED** [review]

File: src/lib/server/editors/PersonaEditor.ts (hunk 1 above)
Comment (Sparkier) [reply src/lib/server/editors/PersonaEditor.ts]: good catch

**Pull request merged**